velox/functions/prestosql/GeometryFunctions.h

```diff
@@ -542,6 +557,58 @@ struct StYFunction {
   }
 };
 
+template <typename T>
+struct StXMinFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE Status
+  call(out_type<double>& result, const arg_type<Geometry>& geometry) {
+    return computeFromEnvelope(
+        result, geometry, [](const geos::geom::Envelope* env) {
+          return env->getMinX();
+        });
+  }
+};
```

Comment:
This this can return `null`, we actually need to:

1. Change the return type to `bool`, `true` indicating non-`null` (valid) and `false` indicating `null`.
2. Use VELOX_USER_FAIL or VELOX_FAIL instead of returning status.